packages/eslint-config-mavenlint/index.js

```diff
@@ -20,6 +19,5 @@ module.exports = {
     'no-console': 'error',
     'no-multiple-empty-lines': ["error", { "max": 1 }],
     'no-use-before-define': ["error", { "functions": false, "classes": true }],
-    'jasmine/no-focused-tests': 'error',
```

Comment:
Decided to remove jasmine config from Mavenlint. Some of our other projects use Karma/Jasmine, and some use Jest. Made sense to me to not make an assumption here about what test framework is used, and I'll add these explicitly back to the Big Maven configuration.

Reply:
Makes sense to me. Out of curiosity, which projects use Jest? I haven't encountered it in Mavenlink yet myself.

Reply:
Ok cool. I was just curious. I don't see choice of unit-testing framework being that big of a deal either way, so this makes sense to me.